_rules/block-collapsible-3e12e1.md

```diff
@@ -0,0 +1,315 @@
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS styling
+acknowledgements:
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectations
+
+For each [section of repeated content][] in the test target, there exists some [user interface component][] which:
+
+- is [visible][]; and
+- is [included in the accessibility tree][]; and
+- allows to toggle both [visibility][visible] and [inclusion in the accessibility tree][included in the accessibility tree] of this [section of repeated content][].
+
+**Note**: the same [user interface component][] may be used for several or even all the [sections of repeated content][section of repeated content].
+
+**Note**: [Technique SCR28: Using an expandable and collapsible menu to bypass block of content][tech scr28] does not require the [accessible name][] of the [user interface component][] to be descriptive. However, having a non-descriptive [accessible name][] is likely a failure of [Success Criterion 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value).
+
+## Assumptions
+
+This rule assumes that [sections of repeated content][section of repeated content] have already been identified within the test target, for example by comparison with other test targets within the same website, or any other means.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [SCR28: Using an expandable and collapsible menu to bypass block of content][tech scr28]
+
+## Test Cases
+
+**Note**: The text of the examples is from the translation of the first Chapter of _The Three Kingdoms_ by Yu Sumei (Tuttle publishing, May 2014).
+
+**Note**: Unless specified otherwise, the [sections of content][] of each document are defined by the [landmarks][landmark] (`aside`, `main` and `nav` elements); both the complementary and navigational [sections of content][] (`aside` and `nav` elements) are [sections of repeated content][section of repeated content].
+
+### Passed
+
+#### Passed Example 1
+
+The [visibility][visible] and [inclusion in the accessibility tree][included in the accessibility tree] of the navigational [section of repeated content][] can be toggled on and off by the link at the start of the document.
+
+```html
+<html>
+	<head>
+		<script src="../test-assets/bypass-blocks-cf77f2/toggle-display.js"></script>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="#" onclick="toggleHidden('navigation')">Toggle table of content</a>
+		<nav id="navigation">
+			<h1>Contents</h1>
+			<!-- list of links to each chapter, repeated on each page -->
+		</nav>
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span of
```

Comment:
Why is this linebreak here in every example?

Reply:
That's HTML prettier cutting lines at 120 characters (iirc).
I agree that it's super ugly in that case, but there is not much we can do about it…